### DISCUSSION TITLE
Test bench maybe should support Service Software and not only public "API endpoint"?

### DISCUSSION THREAD

**Discussion by thadguidry** [reconciliation-api/specs#181] [Ideas]
Browsing through https://github.com/topics/openrefine I was able to find a LOT of reconcile services that are not public endpoints, but self-hosted, service software to spin-up-your-own such as this example:  https://github.com/judaicadh/discogsreconciliation/blob/main/discogs.py

Looking at the test bench dialog for "Add a service" button, I found this:
> Add a statement on that item, using the API endpoint (P6269) property, adding the OpenRefine reconciliation service API (Q64490175) as protocol (P2700) in a qualifier. You can also add a link to the documentation for the endpoint by adding a user manual link (P2078) or described at URL (P973) qualifier. See this example for how it is done.

I think it would be nice to be able to add Service Software, and not only public "API endpoint" 's?  Thoughts?

**Comment by wetneb** [comment]
I'd rather list them separately, for instance in the census, since they can't be tested directly in the test bench.
Beyond the test bench (whose target audience is developers working on reconciliation services and spec authors), there is interest in having a user-facing catalogue of reconciliation services and perhaps self-hostable services could be advertised there.
